docs/securelaunch/README.md

```diff
@@ -0,0 +1,120 @@
+**Note**: Policy file is a *JSON* file, and there is no support for comments in a JSON file.
+***
```

Comment:
remove this? it looks weird when rendered

Reply:
Okay.